pkg/kubelet/kuberuntime/kuberuntime_manager_test.go

```diff
@@ -2745,3 +2747,115 @@ func TestGetImageVolumes(t *testing.T) {
 		assert.Equal(t, tc.expectedImageVolumePulls, imageVolumePulls)
 	}
 }
+
+// This test focuses on verifying `doPodResizeAction()` propagates an error correctly with a few error cases for PR #127300.
```

Comment:
```suggestion
// This test focuses on verifying `doPodResizeAction()` propagates an error correctly with a few error cases.
```

No need to add the PR context, that's what git blame is for